scripts/avlm/avlm_generate.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
do you want to call it avlm in "avlm_generate"? is there a model code name for the supported model?